### PR TITLE
Enable tabs container

### PR DESCRIPTION
**Pull request by Hyperkid123**
### TODO
- [x] sortable header
- [x] add new tab
- ~~[ ] remove tab~~
- [x] sub container selection
- [x] add item to tab
- [x] edit tab properties
- [x] switch active tab
- [x] Initial styling (so it is usable not pretty)

Removing does not work on properly anything yet

tables

### CODE REVIEW

File: src/dnd-kit/ItemsRenderer.js [hunk 1]
```diff
@@ -87,7 +87,11 @@ function ItemsRendererConnector({ items }) {
 }
 
 ItemsRendererConnector.propTypes = {
-  items: PropTypes.arrayOf(PropTypes.string).isRequired,
+  items: PropTypes.arrayOf(PropTypes.string),
+};
+
+ItemsRendererConnector.defaultProps = {
+  items: [],
```
Comment (rvsia) [inline src/dnd-kit/ItemsRenderer.js:94]: You also defined this in the function.

**Review by rvsia — COMMENTED** [review]
- it fails when an item is moved outside tabs
- I cannot move with tabs

**Comment by Hyperkid123** [comment]
> it fails when an item is moved outside tabs

@rvsia  did you move the item all the way to the orange container or just to the first white one?

**Comment by Hyperkid123** [comment]
@rvsia the dragging the whole tabs works but its a bit jank right now. We will fix that once we tweak the collision detection a bit.

janky-drag

I will also prevent dropping stuff directly into the overarching container.

**Comment by Hyperkid123** [comment]
@rvsia can you test it again? Dropping outside should be now OK and dragging "works".

**Comment by rvsia** [comment]
@Hyperkid123 works well now! thanks!

**Review by rvsia — APPROVED** [review]

**Pull request merged**